packages/botonic-cli/tests/analytics/telemetry.test.ts

```diff
@@ -0,0 +1,85 @@
+import { assert } from 'console'
+
+import { Telemetry } from '../../src/analytics/telemetry'
+import { pathExists } from '../../src/util/file-system'
+
+process.env.BOTONIC_DISABLE_ANALYTICS = '1'
+assert(process.env.BOTONIC_DISABLE_ANALYTICS === '1')
```

Comment:
why is there an assert after setting the variable?

Reply:
not needed rewriting the variable, as it is already in our secrets, leaving the assert in order to catch that this is not executed if we have analytics enabled.